tools/c7n_gcp/c7n_gcp/resources/dns.py

```diff
@@ -49,3 +51,23 @@ def get(client, resource_info):
             return client.execute_query(
                 'get', {'project': resource_info['project_id'],
                         'policy': resource_info['name']})
+
+
+@resources.register('dns-project')
+class DNSProject(QueryResourceManager):
```

Comment:
Whats the use case for this resource, its read only...

Reply:
@jonathana, @jondavidi - your assistance with a use case will be appreciated.

Reply:
@kapilt, agreed to remove the resource, thank you.